perfzero/docker/Dockerfile_ubuntu_1804_s4tf_cuda11.0

```diff
@@ -0,0 +1,131 @@
+# Ubuntu 18.04 Python3 with CUDA 11.0 and the following:
+#  - Installs tf-nightly-gpu (this is TF 2.4)
+#  - Installs requirements.txt for tensorflow/models
+# Additionally also installs
+#  - Latest S4TF development snapshot for cuda 11.0
+
+FROM nvidia/cuda:11.0-base-ubuntu18.04 as base
+ARG tensorflow_pip_spec="tf-nightly-gpu"
+ARG local_tensorflow_pip_spec=""
+ARG extra_pip_specs=""
+ARG swift_tf_url=https://storage.googleapis.com/swift-tensorflow-artifacts/nightlies/latest/swift-tensorflow-DEVELOPMENT-cuda11.0-cudnn8-ubuntu18.04.tar.gz
+
+# setup.py passes the base path of local .whl file is chosen for the docker image.
+# Otherwise passes an empty existing file from the context.
+COPY ${local_tensorflow_pip_spec} /${local_tensorflow_pip_spec}
+
+# Pick up some TF dependencies
+# cublas-dev and libcudnn8-dev only needed because of libnvinfer-dev which may not
+# really be needed.
+# In the future, add the following lines in a shell script running on the
+# benchmark vm to get the available dependent versions when updating cuda
+# version (e.g to 10.2 or something later):
+# sudo apt-cache search cuda-command-line-tool
+# sudo apt-cache search cuda-cublas
+# sudo apt-cache search cuda-cufft
+# sudo apt-cache search cuda-curand
+# sudo apt-cache search cuda-cusolver
+# sudo apt-cache search cuda-cusparse
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        cuda-command-line-tools-11-0 \
+        cuda-cufft-10-2 \
+        cuda-curand-10-2 \
+        cuda-cusolver-10-2 \
+        cuda-cusparse-10-2 \
```

Comment:
For these, can we instead use cuda-tools-11-0 and cuda-toolkit-11-0, like is done here: https://github.com/tensorflow/benchmarks/blob/master/perfzero/docker/Dockerfile_ubuntu_1804_tf_cuda_11#L29 ?

Reply:
Got it. Will try. 

Reason I use 10.2 is because I found them from https://docs.nvidia.com/cuda/archive/11.0/cuda-toolkit-release-notes/index.html#major-components. Whereas actually cusolver is 10.6, cusparse is 11.1, but I can't make it work with that.

Reply:
Done